configs/katello/3.9.yaml

```diff
@@ -0,0 +1,270 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+  :3.8.0:
+    :redmine_version_id: 356
+  :3.7.0:
+    :redmine_version_id: 264
+  :3.7.1:
+    :redmine_version_id: 355
+:releases:
+  :3.9.0:
+    :redmine_version_id: 899
+:code_name: New England IPA
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-3.8
```

Comment:
3.9?

Reply:
It's not there yet, so I hadn't put it in.

Reply:
I'll take it out for now.

Reply:
Good prep PR -- I think setting it what it *will* be is good (e.g. KATELLO-3.9) vs. removing